src/smhi/metobs.py

```diff
@@ -306,7 +306,7 @@ class Data(BaseMetobs):
     def __init__(
         self,
         periods_in_station: Periods,
-        period: str = "corrected-archive",
+        period: str = "not-set",
```

Comment:
What does this mean?

Reply:
You are using the same construct in `smhi.py`. It means that `period` is optional, i.e. that it can be of type `str` or `None` (if not provided, hence the optional).

Reply:
Ok, but if I put it as ```None```, won't that screw my logic for testing whether the choice actually exists and if it isn't provided set latest-whatever-exists as the period? Shouldn't it be ```period: Optional[str] = "not-set"```?

Reply:
You can instead check

```python
if period is None:
    ...
```